launchpad-services/configmapcontroller.yaml

```diff
@@ -0,0 +1,5 @@
+services:
+- hash: db0d424c677142d4f152075f4f968e8c990881a6
```

Comment:
This should be 971412133b88b47660c33cb77a6ec1fd4a392558

Reply:
why?

Reply:
As in https://github.com/fabric8-launcher/launcher-openshift-templates/commit/971412133b88b47660c33cb77a6ec1fd4a392558

Reply:
ah because it's the hash it will use to check out that file? ok

Reply:
Just guessing, as that is the latest commit ID in the repo. The Hash you provided doesn't seem to have the file in the new repository name change path